packages/typo3-guides-cli/tests/unit/Migration/SettingsMigratorTest.php

```diff
@@ -35,9 +35,9 @@ public static function providerForMigrateReturnsXmlDocumentCorrectly(): \Generat
         yield 'with empty legacy settings' => [
             'legacy settings' => [],
             'expected' => <<<EXPECTED
-<?xml version="1.0" encoding="UTF-8"?>
-<guides xmlns="https://www.phpdoc.org/guides" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd" links-are-relative="true"/>
-EXPECTED,
+                <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Doesn't this change the expected output due to introduced whitespace?

Reply:
Nope, as we assert XML string with XML string in the unit tests.